src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.stories.js

```diff
@@ -0,0 +1,25 @@
+import Carousel from "nuka-carousel";
+import React from "react";
+import BlockQuoteBlock from "./BlockQuoteBlock";
+
+export default {
+  title: "Grid-Aware/BlockQuoteBlock",
+  component: BlockQuoteBlock,
+};
+
+const Template = ({ quote, author }) => (
+  <Carousel renderCenterLeftControls={false} renderCenterRightControls={false}>
```

Comment:
Looks like `false` is not a valid value to put here, since it's expecting a [render prop](https://reactjs.org/docs/render-props.html), that is, a function that returns a JSX component in the end. Maybe you could just try passing in an empty function?

Reply:
👍 Passed in empty functions and worked. Will you please explain why inserting empty functions will work?

Reply:
The nuka-carousel [documentation](https://github.com/FormidableLabs/nuka-carousel#rendercontrols) said that these parameters take in render props. If you look at the link that I posted in my previous message, you'll see that a render prop is a prop that takes in a function and return a React component. I assumed that you could pass in a render prop argument that returned nothing in order to not actually render anything.

However, now that I read the nuka-carousel documentation more closely, I see that you're actually just supposed to pass in `null` if you don't want these to render:

> To change the position or remove "Paging dots", the default positions need to be disabled by setting them to null.

Reply:
👍 We talked about this and changed to pure-react-carousel instead of using nuka-carousel.